lib/meandro/rules/unused_callbacks.ex

```diff
@@ -66,7 +66,7 @@ defmodule Meandro.Rule.UnusedCallbacks do
   # We need to keep track of the current module as well, to avoid nested modules messing up
   # our assumptions.
   defp collect_callbacks(
-         {:callback, _, _} = callback,
+         {:callback, _, [{:"::", _, _}]} = callback,
```

Comment:
please comment

Reply:
A variable is represented in AST as `{:variable_name, meta_info, nil`}, and when the variable happened to be called `callbacks`, it would match on this function, and thus cause some issues down the call stack.

This change in the function clause is to ensure we only match on actual callback definitions.

Reply:
I meant in the code!